.drone.yml

```diff
@@ -902,8 +902,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-amd64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
```

Comment:
I thought we use manifest to create latest tag:
https://github.com/go-gitea/gitea/blob/main/docker/manifest.tmpl#L6-L8

Reply:
the auto_tag is what is putting those there.